### DISCUSSION TITLE
视频缓冲速度变慢，连续快进有时会自动暂停

### DISCUSSION THREAD

**Discussion by BJDSYC** [the1812/Bilibili-Evolved#3197] [提问 / Q&A]
感觉应该是今天才遇到的，开了脚本快进的时候缓冲速度明显变慢，经常卡顿需要等几秒，连续快进有时会自动卡住，还需要手动点播放，关闭脚本就正常了，是bug还是我的电脑问题？

**Comment by sky-279** [comment]
同样的问题 开了脚本后网速明显变慢 稍微快进就显示正在缓冲但是没网速 已经使用centbrowser和edge分别做过测试 已排除浏览器和网络问题
